MaterialDesignThemes.Wpf/CustomBaseColorTheme.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System.Windows;
+using System.Windows.Media;
+
+namespace MaterialDesignThemes.Wpf
+{
+    public class CustomBaseColorTheme : ResourceDictionary, IBaseTheme
+    {
+        public bool DesignTime { get; set; } = false;
```

Comment:
Is this needed? I see that it prevents calls to `SetTheme()` but given than this resource dictionary shouldn't be loaded anywhere, I would not expect that to be a problem.

Reply:
In fact, I removed this snippet, and I didn't notice any problems.

I imagined at the beginning that I should treat the Theme Editor in a different way by changing the Primary and Secondary color values, preventing an unwanted color change throughout the DemoApp.

However, in practice, this has not occurred. Removing ... Thanks.